vogue/server/endpoints/common_trends.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python
+
+from flask import render_template, Blueprint, current_app, request
+
+from vogue.constants.constants import MONTHS, YEARS, THIS_YEAR
+from vogue.server.utils.metric_per_month import value_per_month
+from vogue import __version__
+
+app = current_app
+common_trends_blueprint = Blueprint('common_trends', __name__)
+
+
+@common_trends_blueprint.route('/common/turn_around_times/<year>')
+def turn_around_times(year):
```

Comment:
I'm going to push for type hints everywhere as it gets easier to read the code, easier to program the code and easier to avoid bugs
```suggestion
def turn_around_times(year: int):
```